collect_app/src/main/java/org/odk/collect/android/formmanagement/formmap/FormMapViewModel.kt

```diff
@@ -194,7 +194,7 @@ class FormMapViewModel(
 
     private fun getDrawableIdForStatus(status: String, enlarged: Boolean): Int {
         return when (status) {
-            Instance.STATUS_INCOMPLETE -> if (enlarged) R.drawable.ic_room_form_state_incomplete_48dp else R.drawable.ic_room_form_state_incomplete_24dp
+            Instance.STATUS_INCOMPLETE, Instance.STATUS_VALID, Instance.STATUS_INVALID -> if (enlarged) R.drawable.ic_room_form_state_incomplete_48dp else R.drawable.ic_room_form_state_incomplete_24dp
```

Comment:
We might want to distinguish somehow valid and invalid instances but that's something for a separate issue that could be a part of v2023.4